### PR TITLE
Fix SWMM summary tables

### LINKED ISSUE #141: SWMM_MTP4-r2_Summary report tables
Some of the summary report tables have errors.

* Subcatchment runoff: runoff coefficient should be unitless.  In metric units, wrong units also for total runoff and peak runoff.

* Groundwater: has incorrect parameters.  

* Node inflow: sometimes has an extra column.

* Node surcharge: appears even when no surcharge occurs.

* Node flooding: appears even when no flooding occurs.  Has incorrect parameters.  

* Storage volume: missing units for average volume, maximum volume, wrong units for maximum outflow.

* Outfall loading: has incorrect parameters.  

* Conduit surcharge: appears even when no conduits surcharge.  If there are link pollutants, it may show the values that appear in the link pollutant load table.

* Flow classification: can appear when it shouldn't.

### PR DESCRIPTION
**Pull request by barrc**
Fixes for SWMM summary report tables.  Closes #141.

### CODE REVIEW

**Comment by michaeltryby** [comment]
@barrc Its great that you are digging in on this! Since the team is most likely going to be you, me, and @woohn, we should be reviewing each others work. In the future assign us as reviewers. Sound good?

**Comment by barrc** [comment]
@michaeltryby sounds good!

File: src/ui/SWMM/frmSummaryReport.py [hunk 1]
```diff
@@ -43,6 +43,15 @@ def set_from(self, project, status_file_name):
...
             try:
+                with open(status_file_name, 'r') as inp_reader:
+                    for line in iter(inp_reader):
+                        if line.startswith('  No nodes were surcharged.'):
+                            potential_topics.remove('Node Surcharge')
+                        elif line.startswith('  No nodes were flooded.'):
+                            potential_topics.remove('Node Flooding')
+                        elif line.startswith('  No conduits were surcharged.'):
+                            potential_topics.remove('Conduit Surcharge')
+
```
Comment (michaeltryby) [inline src/ui/SWMM/frmSummaryReport.py]: This looks good!

File: src/ui/SWMM/frmSummaryReport.py [hunk 2]
```diff
@@ -227,9 +236,14 @@ def cboType_currentIndexChanged(self, newIndex):
                                 if header_count == total_num_headers:
                                     if temp_header_list[1] == 'Node':
                                         temp_header_list.remove('Node')
-                                    if temp_header_list[i+1] == '10^6' and temp_header_list[i+2] == 'gal':
-                                        temp_header_list[i+1] = '10^6 gal'
+                                    if temp_header_list[i+1] == '10^6':
+                                        if temp_header_list[i+2] == 'gal':
+                                            temp_header_list[i+1] = '10^6 gal'
+                                        elif temp_header_list[i+2] == 'ltr':
+                                            temp_header_list[i+1] = '10^6 ltr'
```
Comment (michaeltryby) [inline src/ui/SWMM/frmSummaryReport.py]: It would be good to think about a more comprehensive approach for handling units and unit conversions.

**Comment by michaeltryby** [comment]
We should also think about adding unit tests as we go, to insure that things we fix are actually fixed / stay fixed. Make sense?

**Comment by michaeltryby** [comment]
@barrc Hold off on merging for the moment. Ok. I don't like to work directly in master.

File: src/ui/SWMM/frmSummaryReport.py (hunk 2 above)
Comment (barrc) [reply src/ui/SWMM/frmSummaryReport.py]: Yeah, I agree.

Comment (barrc) [reply src/ui/SWMM/frmSummaryReport.py]: I altered the check slightly but recognize that it's still not a comprehensive approach.

**Pull request merged**